### PR TITLE
Feature/tr 1867/launch test with lti 1p3

### PR DESCRIPTION
**Pull request by emgolubev**
Related to: https://oat-sa.atlassian.net/browse/TR-1867

Test:
- you can use https://github.com/oat-sa/nextgen-stack/tree/feature/TR-1577/oidc-initiation for easier testing

Example

http://lti.docker.localhost/platform/message/launch/lti-resource-link?registration=devkitToTAO&user_list=c3po&launch_url=http://backoffice.docker.localhost/ltiDeliveryProvider/DeliveryTool/launch1p3/?delivery%3Dhttp%253A%252F%252Fbackoffice.docker.localhost%252Fontologies%252Ftao.rdf%2523i613888211402388794933c0190e372&claims=%7B%0D%0A%20%20%20%20%22https://purl.imsglobal.org/spec/lti/claim/roles%22:%20%5B%0D%0A%20%20%20%20%20%20%20%20%22http://purl.imsglobal.org/vocab/lis/v2/membership%23Learner%22%0D%0A%20%20%20%20%5D%0D%0A%7D

### CODE REVIEW

File: controller/DeliveryTool.php [hunk 1]
```diff
@@ -182,6 +183,14 @@ public function checkCapacity()
             ->withHeader('Content-Type', 'application/json');
     }
 
+    public function launch1p3(): void
+    {
+        $message = $this->getValidatedLtiMessagePayload();
+
+        LtiService::singleton()->startLti1p3Session($message);
```
Comment (wazelin) [inline controller/DeliveryTool.php:190]: Don't forget to update `composer.json` with a new version of `extension-tao-lti`, implementing that method.
Feel free to specify the `dev-*` version there so far, just as a placeholder, not to forget bumping the requirement before the merge.

**Review by wazelin — CHANGES_REQUESTED** [review]

**Review by alexeyvy — APPROVED** [review]
- [ ] New code is covered by tests (if applicable) - not applicable, as the changes are places in controllers/migration/manigest - therefore, not testable in terms of cgen
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by wazelin — CHANGES_REQUESTED** [review]
I can see a few empty files here.
Also, the file permissions seem to have changed for some reason.

image

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Comment by wazelin** [comment]
Please merge only after the taoLti extension has been released by the automated GitHub action.

**Pull request merged**